pwem/objects/data.py

```diff
@@ -1579,8 +1579,11 @@ def scale(self, factor):
         m *= factor
         m[3, 3] = 1.
 
-    def scaleShifts(self, factor):
+    def scaleShifts(self, factor, shiftsAppliedBefore=False):
         m = self.getMatrix()
+        if shiftsAppliedBefore:
+            m[0, 3] = m[0, 3] - int(m[0, 3])  # Decimal part of X translation
```

Comment:
I've found this other way to get the decimal part, I like more. But there is no need to change it.

5.55 % 1 --> 0.55

Reply:
I like it, it's more elegant and concise.